metadata.json

```diff
@@ -18,11 +18,11 @@
   "requirements": [
     {
       "name": "pe",
-      "version_requirement": "3.x"
+      "version_requirement": ">= 3.4.0 < 2015.3.0"
     },
     {
       "name": "puppet",
-      "version_requirement": ">= 3.3.0 < 4.0.0"
+      "version_requirement": ">= 3.3.0 < 5.0.0"
```

Comment:
There was a mistake in a release that changed this value and it needs to be fixed.